src/Cli/dotnet/commands/dotnet-sln/remove/Program.cs

```diff
@@ -47,7 +47,7 @@ public override int Execute()
                             ? MsbuildProject.GetProjectFileFromDirectory(fullPath).FullName
                             : fullPath);
                 });
-                RemoveProjectsAsync(solutionFileFullPath, relativeProjectPaths, CancellationToken.None).Wait();
+                RemoveProjectsAsync(solutionFileFullPath, relativeProjectPaths, CancellationToken.None).GetAwaiter().GetResult();
```

Comment:
I remember it being bad (potential deadlocks) to use `GetAwaiter().GetResult()`. But since we don't have an `ExecuteAsync`, I don't think there is another option.

Reply:
Yes, I updated it to be the same as `sln add`, and to better catch the exceptions